### PR TITLE
bypass state_estimation for now to allow testing

### PR DESCRIPTION
**Pull request by VasilyF**

### CODE REVIEW

**Comment by antoinedang** [comment]
@MrMondrian @frompotenza can we close this PR?

**Comment by MrMondrian** [comment]
No I think during comp we all were using the bypass branch. I bet noetic is still broken

**Comment by MrMondrian** [comment]
also shouldn't the main branch for this repo be fortress?

**Comment by antoinedang** [comment]
> also shouldn't the main branch for this repo be fortress?

@frompotenza

**Comment by antoinedang** [comment]
> No I think during comp we all were using the bypass branch. I bet noetic is still broken

Should I merge it then?

**Comment by MrMondrian** [comment]
If you test it then sure

**Review by MrMondrian — APPROVED** [review]

**Comment by frompotenza** [comment]
> also shouldn't the main branch for this repo be fortress?

it is fortress (?)

File: src/bridge.py [hunk 1]
```diff
@@ -32,25 +32,36 @@ def cb_sim_pose(data):
...
+            if angles[i] - euler[i] > ANGLE_CHANGE_TOL:
+                euler[i] = angles[i] - 360
+            elif euler[i] - angles[i] > ANGLE_CHANGE_TOL:
+                euler[i] = angles[i] + 360
+            else:
+                euler[i] = angles[i]
+
+    pub_state_theta_x.publish(euler[0])
+    pub_state_theta_y.publish(euler[1])
+    pub_state_theta_z.publish(euler[2])
```
Comment (frompotenza) [inline src/bridge.py:64]: Would you prefer the main branch to be the one that bypasses state estimation and a secondary one that doesn't bypass or the other way around?

File: src/transform_broadcaster_sim.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+import rospy
+
+from tf2_ros import TransformBroadcaster, StaticTransformBroadcaster
+from geometry_msgs.msg import TransformStamped, Pose
+
+'''
+Broadcasts the current coordinate transformation of the auv with respect to the world frame.
+Listens to the pose topic to get the transformation, then broadcast it as a tf2 transformation.
+The world frame is north east up.
```
Comment (frompotenza) [inline src/transform_broadcaster_sim.py:11]: shouldn't it be north west up?

**Review by frompotenza — DISMISSED** [review]

**Comment by MrMondrian** [comment]
Yes it should, maybe it’s just a typo

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: src/bridge.py (hunk 1 above)
Comment (antoinedang) [reply src/bridge.py:64]: whichever one works should be main branch, in the future ideally we want to not need to bypass at all

**Comment by antoinedang** [comment]
> If you test it then sure

Since we're re-doing state estimation and this is the branch that we've been using basically as a main branch I'm gonna merge, we can integrate the sim into state estimation once state estimation has been re-done and is working. Lmk if you agree @frompotenza @MrMondrian

**Pull request merged**